examples/project_dagster_university_start/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.dagster]
+module_name = "dagster_university"
```

Comment:
NTS on next push: update this and other references to `project_dagster_university`